packages/react-vapor/src/components/actions/examples/ActionBarExamples.tsx

```diff
@@ -71,6 +71,35 @@ export class ActionBarExamples extends React.Component<any, any> {
                 <Section level={2} title="Action bar disabled">
                     <ActionBar actions={actions} disabled />
                 </Section>
+                <Section level={2} title="Action bar disabled with a secondary action converted to a primary action">
+                    <ActionBar
+                        actions={[
+                            {
+                                name: 'Link to Coveo (disabled)',
+                                link: 'http://coveo.com',
+                                target: '_blank',
+                                icon: 'exit',
+                                primary: true,
+                                enabled: false,
+                                hideDisabled: false,
+                                tooltip: "You cannot access Coveo's website at the moment.",

```

Comment:
`tooltip: 'You cannot access Coveo's website at the moment.',`
pour les autres strings, on utilise ' ' au lieu de " "